### ISSUE TITLE
MD013: Add override for lists

### ISSUE THREAD

**Issue by lonix1** [DavidAnson/markdownlint#930]
[MD013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md) has overrides for tables, headings, code blocks, etc.

But not for lists:
```md
- hello
- world
- asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf
- foo bar baz
```

It's common to have a paragraph as a bullet point. (Regrettably github doesn't render it nicely above, but in other tools it would be wrapped.)

That list inherits the 80 char limit. It would be nice to break the rule for lists, without changing the 80 value (which is good, in general).

**Comment by DavidAnson** [comment]
The exceptions you list exist mainly because it may not be possible to limit their width. For example, table rows need to be on a single line or else it breaks the table structure. However, that is not the case for list items which behave the same as normal text. If someone wants to limit text to 80 columns, I don't see why they would want to allow list items to extend further when they could be wrapped at 80 columns just as easily. What's the scenario you have in mind here?

**Comment by DavidAnson** [comment]
I don't think I understand. The following produces a three-item list with wrapping at 14 (or so) columns. I expect a similar approach to work for most lists.

```md
- One
- Two two two
  two two two
  two two
- Three
```
